docs/Contributing/rituals/weekly-estimation.md

```diff
@@ -0,0 +1,38 @@
+## [Weekly Estimation](https://fleetdm.com/handbook/company/product-groups#sprint-ceremonies) 
+
+Purpose: Review user stories and bugs that have completed the drafting and specification process and add point estimates.
+
+Cadence: Weekly, 1 hour or more as needed.
+
+Participants: Full product group.
+
+Facilitator: EM or assigned team member.
+
+> All participants are expected to review every user story for their product group in the "Ready to estimate" column before Weekly Estimation. It is the facilitator's responsibility to make sure the team is notified and has adequate time to review each issue before the ritual.
+
```

Comment:
+1 on this, and finding a way to ritualize this (perhaps it's a bullet point for the EM to mention this in the daily standup?). Having stories specced out in a standardized way before they get to "ready to estimate" will help here, since a story is easier to estimate when the requirements and subtasks are clear. Using async estimates gives us a built-in place (the poker planner thread) to have discussions prior to the estimation meeting, but if we aren't mandating async estimation then we should consider where else these discussions should take place.

Reply:
Agreed, if planning isn't async then the EM needs to be proactive about starting a thread and tagging his team when one or more user stories are ready to review for estimation.